addons/mail/static/src/core/common/message.xml

```diff
@@ -177,4 +178,10 @@
     </div>
 </t>
 
+<t t-name="mail.Message.notification.body">
+    <div class="o-mail-Message-body text-break mb-0 w-100">
+        <t t-out="messageSearch?.highlight(message.body) ?? message.body"/>
```

Comment:
Uh-oh... `messageSearch` doesn't exist... `props.messageSearch` yes...
This looks like a bug 😵

Reply:
I'm gonna check it in another PR :)